timelapse.py

```diff
@@ -67,36 +67,45 @@ def capture_image():
         set_camera_options(camera)
 
         # Capture a picture.
-        camera.capture(dir + '/image{0:05d}.jpg'.format(image_number))
+        camera.capture(dir + '/image{0:08d}.jpg'.format(image_number+1))
```

Comment:
The one change I think I'm not in favor of is starting at 1... though someone may be able to make a good argument for it. I like 0-indexed, as that makes dealing with the series easier later.

Reply:
ok, works for me, i just started to mess around timelapses, so probably there is a good reason it starts from 0.